src/documentation/0019-package-json/index.md

```diff
@@ -8,7 +8,7 @@ If you work with JavaScript, or you've ever interacted with a JavaScript project
 
 What's that for? What should you know about it, and what are some of the cool things you can do with it?
 
-The `package.json` file is kind of a manifest for your project. It can do a lot of things, completely unrelated. It's a central repository of configuration for tools, for example. It's also where npm and `yarn` store the names and versions of the package it installed.
+The `package.json` file is kind of a manifest for your project. It can do a lot of things, completely unrelated. It's a central repository of configuration for tools, for example. It's also where npm and `yarn` store the names and versions of the package they installed.
```

Comment:
```suggestion
The `package.json` file is kind of a manifest for your project. It can do a lot of things, completely unrelated. It's a central repository of configuration for tools, for example. It's also where `npm` and `yarn` store the names and versions of the packages.
```

Reply:
@jgb-solutions I suggest you merge the suggestion from @mbj36 so I can merge this PR. 👍